packages/scripts/templates/block-plugin/plugin.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Plugin Name: block_plugin_name
+ * Version: 1.0.0
+ * License: GPL2+
+ * License URI: https://www.gnu.org/licenses/gpl-2.0.txt
+ *
+ * @package block_plugin_name
+ */
+
+/**
+ * Enqueue editor and front end assets.
+ *
+ * @since 1.0.0
+ */
+function block_plugin_name_enqueue_block_assets() {
```

Comment:
Testing this PR
> `Uncaught Error: syntax error, unexpected '-', expecting '(' in /wp-content/plugins/my-block-plugin/plugin.php on line 16`

The function name is being generated as:

> `my-block-plugin_enqueue_block_assets()`

Instead of kebabCase, the `my-block-plugin` needs to be snakeCase:

> `my_block_plugin_enqueue_block_assets()`

Reply:
Oops, should be fixed now. Thanks for catching this!